hack/verify-all.sh

```diff
@@ -34,37 +34,37 @@ fi
 
```

Comment:
Didn't move verify-all.sh yet since the prow presubmit still points to this.

Reply:
we can make this alias the actual script and then update the job, come back and remove the alias